### PR TITLE
docs: v2 event-types

### PR DESCRIPTION
**Pull request by supalarry**
Linear [CAL-4424](https://linear.app/calcom/issue/CAL-4424/platform-docs-event-types)

### CODE REVIEW

File: packages/platform/types/event-types/event-types_2024_06_14/inputs/update-event-type.input.ts [hunk 1]
```diff
@@ -262,11 +317,16 @@ export class UpdateEventTypeInput_2024_06_14 {
     return value;
   })
   @ValidateNested()
+  @DocsPropertyOptional({
+    description: "Create a recurring event type.",
```
Comment (SomayChauhan) [inline packages/platform/types/event-types/event-types_2024_06_14/inputs/update-event-type.input.ts:321]: needs an update
same for `create-event-type.input.ts`

File: packages/platform/types/event-types/event-types_2024_06_14/inputs/update-event-type.input.ts [hunk 2]
```diff
@@ -150,77 +166,107 @@ export class UpdateEventTypeInput_2024_06_14 {
...
 
   @IsOptional()
   @ValidateBookingWindow()
-  @DocsProperty({
+  @DocsPropertyOptional({
     description: "Limit how far in the future this event can be booked",
     oneOf: [
       { $ref: getSchemaPath(BusinessDaysWindow_2024_06_14) },
       { $ref: getSchemaPath(CalendarDaysWindow_2024_06_14) },
       { $ref: getSchemaPath(RangeWindow_2024_06_14) },
```
Comment (SomayChauhan) [inline packages/platform/types/event-types/event-types_2024_06_14/inputs/update-event-type.input.ts:235]: add `Disabled_2024_06_14` as well
same for `create-event-type.input.ts`

**Review by ibex088 — CHANGES_REQUESTED** [review]

File: packages/platform/types/event-types/event-types_2024_06_14/inputs/update-event-type.input.ts (hunk 2 above)
Comment (supalarry) [reply packages/platform/types/event-types/event-types_2024_06_14/inputs/update-event-type.input.ts:235]: Well spotted! Implemented in [refactor: add disabled option to booking window](https://github.com/calcom/cal.com/pull/16854/commits/75fb06324893893889e6e3e7f0d98335e195099d)

File: packages/platform/types/event-types/event-types_2024_06_14/inputs/update-event-type.input.ts (hunk 1 above)
Comment (supalarry) [reply packages/platform/types/event-types/event-types_2024_06_14/inputs/update-event-type.input.ts]: Fixed in [refactor: seats description](https://github.com/calcom/cal.com/pull/16854/commits/c5dde562abde0703e56ea01ffa70543fc42529fb)

**Review by ibex088 — APPROVED** [review]
LGTM

**Pull request merged**